src/base/projects/Header.svelte

```diff
@@ -128,49 +102,47 @@
   {/if}
   <span>
     {#if seed.api.host}
-      <!-- svelte-ignore a11y-click-events-have-key-events -->
-      <div
-        class="stat seed clickable widget"
+      <Stat
+        clickable
+        ariaLabel="seed"
         title="Project data is fetched from this seed"
         on:click={goToSeed}>
         <span>{seed.api.host}</span>
-      </div>
+      </Stat>
     {/if}
   </span>
-  <!-- svelte-ignore a11y-click-events-have-key-events -->
-  <div
-    class="stat commit-count clickable widget"
-    class:active={activeRoute.params.view.resource === "history"}
+  <Stat
+    ariaLabel="commit-count"
+    clickable
+    active={activeRoute.params.view.resource === "history"}
     on:click={() => toggleContent("history", true)}>
     <span class="txt-bold">{tree.stats.commits}</span>
     commit(s)
-  </div>
+  </Stat>
   {#if project.issues}
-    <!-- svelte-ignore a11y-click-events-have-key-events -->
-    <div
-      class="stat issue-count clickable widget"
-      class:active={activeRoute.params.view.resource === "issues"}
-      class:not-allowed={project.issues === 0}
-      class:clickable={project.issues > 0}
+    <Stat
+      ariaLabel="issue-count"
+      active={activeRoute.params.view.resource === "issues"}
+      disabled={project.issues === 0}
+      clickable={project.issues > 0}
       on:click={() => toggleContent("issues", false)}>
       <span class="txt-bold">{project.issues}</span>
       issue(s)
-    </div>
+    </Stat>
   {/if}
   {#if project.patches}
-    <!-- svelte-ignore a11y-click-events-have-key-events -->
-    <div
-      class="stat patch-count clickable widget"
-      class:active={activeRoute.params.view.resource === "patches"}
-      class:not-allowed={project.patches === 0}
-      class:clickable={project.patches > 0}
+    <Stat
+      ariaLabel="patch-count"
+      clickable={project.patches > 0}
+      active={activeRoute.params.view.resource === "patches"}
+      disabled={project.patches === 0}
       on:click={() => toggleContent("patches", false)}>
       <span class="txt-bold">{project.patches}</span>
       patch(es)
-    </div>
+    </Stat>
   {/if}
-  <div class="stat contributor-count widget">
+  <Stat ariaLabel="contributor-count">
```

Comment:
Could we spell out them without the hyphen? E.g. "Contributor count".

Reply:
I liked to pass a combined word like `contributor-count` to the test suite since we use it mostly for testing.
But I think it we should be able to spell it out

Reply:
I wouldn't have anything against hyphenating it if it wasn't an accessibility attribute which might be read by screen readers to text-to-speech for people who need it.

Reply:
e7c6607e